ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/cli/Cli.scala

```diff
@@ -137,7 +137,7 @@ object Cli {
             .fold(Some(TlsConfiguration(enabled = true, None, None, None, clientAuth)))(c =>
               Some(c.copy(clientAuth = clientAuth)))))
 
-      opt[Int]("maxInboundMessageSize")
+      opt[Int]("max-inbound-message-size")
```

Comment:
Please only add a deprecation message to the help text and introduce a new option. This is to avoid breaking existing scripts.

Reply:
Done.